src/lib/utils/SynapseClient.ts

```diff
@@ -92,12 +92,17 @@ import { VersionInfo } from './synapseTypes/VersionInfo'
 import { SearchQuery, SearchResults } from './synapseTypes/Search'
 import { ResearchProject } from './synapseTypes/ResearchProject'
 import {
-  ManagedACTAccessRequirementStatus,
   RequestInterface,
```

Comment:
Was there a merge conflict? This line is required for the ManagedACTAccess component.

Reply:
Nice catch, this was a bad merge from conflict resolution